### ISSUE TITLE
Grafana role on monitoring project not working

### ISSUE THREAD

**Issue by nbozic-zz** [smartcat-labs/smartcat-ops#20]
When new data source over api is attempted it fails with 

```
TASK [grafana-docker : Creating Grafana default InfluxDB DataSource] ***********
fatal: [192.168.35.30]: FAILED! => {"changed": false, "content": "[{\"classification\":\"DeserializationError\",\"message\":\"json: cannot unmarshal string into Go value of type models.AddDataSourceCommand\"},{\"fieldNames\":[\"Name\"],\"classification\":\"RequiredError\",\"message\":\"Required\"},{\"fieldNames\":[\"Type\"],\"classification\":\"RequiredError\",\"message\":\"Required\"},{\"fieldNames\":[\"Access\"],\"classification\":\"RequiredError\",\"message\":\"Required\"}]", "content_length": "371", "content_type": "application/json; charset=utf-8", "date": "Wed, 10 Aug 2016 14:12:25 GMT", "failed": true, "json": [{"classification": "DeserializationError", "message": "json: cannot unmarshal string into Go value of type models.AddDataSourceCommand"}, {"classification": "RequiredError", "fieldNames": ["Name"], "message": "Required"}, {"classification": "RequiredError", "fieldNames": ["Type"], "message": "Required"}, {"classification": "RequiredError", "fieldNames": ["Access"], "message": "Required"}], "msg": "Status code was not [200]", "redirected": false, "status": 400}
```

**Comment by nbozic-zz** [comment]
@nivancevic can you check this out when you have time

**Comment by nbozic-zz** [comment]
Closed with #22
